src/GitUtils.ps1

```diff
@@ -415,6 +415,52 @@ function Get-SshAgent() {
     return 0
 }
 
+function Get-NativeSshAgent {
+    # The ssh.exe binary version must include "OpenSSH"
+    # The windows ssh-agent service must exist
+    $service = Get-Service ssh-agent
+    $executableMatches = Get-Command ssh.exe | ForEach-Object FileVersionInfo | Where-Object ProductVersion -match OpenSSH
+    $valid = $service -and $executableMatches
+
+    if ($valid) {
+        return $service;
+    }
+
+    return $false;
```

Comment:
Would it be more PowerShell-y to `return $null` here? Or to not `return` anything at all?

Reply:
Return nothing if the service is not installed.  That will successfully compare to $null.

Reply:
Done.